CodeU_Assignement_3/test/codeu_assignement_3/SimpleDictionaryTest.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package codeu_assignement_3;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Karine
+ */
+public class SimpleDictionaryTest {
+    
+    public SimpleDictionaryTest() {
+    }
+    
+    @BeforeClass
+    public static void setUpClass() {
+    }
+    
+    @AfterClass
+    public static void tearDownClass() {
+    }
+
+    /**
+     * Test of isWord method, of class SimpleDictionary.
+     */
+    @Test
+    public void testIsWordNullDict() {
+        System.out.println("isWord Null dictionary");

```

Comment:
So what can I do instead the description of the test? (line number of the fail?) I'm a bit confused...